.binder/README.md

```diff
@@ -0,0 +1,30 @@
+# Docker Image for Binder
+
+<!--
+This is derived from the following setup:
+https://github.com/RobotLocomotion/drake/tree/dc2a9394d/.binder
+-->
+
+*Note that due to Binder conventions, this directory MUST always be in the root
+of the repository and named either `binder` or `.binder`. This image is NOT
+intended for use by most developers or users.*
+
+To create a Docker image and run a Docker container similar to those used by
+[Binder](https://mybinder.org) for local debugging purposes, execute the
+following `pull`, `build`, and `run` commands from the top level of this Git
+repository:
+
+```bash
+cd pygccxml
```

Comment:
Is this needed? Assuming we're already at the top level of the git repo, this would put is at `pygccxml/pygccxml`. Then `.binder` would be in the parent directory. Excluding this `cd` call works for me

Reply:
Done. Removed.